gui/src/common/unsupported-features.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lyve_pilot": "lyve_pilot"
```

Comment:
with end to end implementation now, this file will not be required

Reply:
Removed this file.